docs/source/getting-started/registering-first-model/step1-register-model.rst

```diff
@@ -0,0 +1,75 @@
+Register a Model
+=================
+
+Throughout this tutorial we will leverage a local tracking server and model registry for simplicity.
+However, for production use cases we recommend using a 
+`remote tracking server <https://mlflow.org/docs/latest/tracking/tutorials/remote-server.html>`_.
+
+Step 0: Install Dependencies
+----------------------------
+.. code-section::
+    .. code-block:: bash
+
+        pip install --upgrade mlflow
+
+Step 1: Register a Model
+--------------------------------
+
```

Comment:
I would add some more high level information. Such as: to use the model registry, you need to add your MLflow models to it. This is done through the registering that model.